modules/@apostrophecms/template/index.js

```diff
@@ -659,6 +659,7 @@ module.exports = {
             tabId: self.apos.util.generateId(),
             uploadsUrl: self.apos.attachment.uploadfs.getUrl(),
             assetBaseUrl: self.apos.asset.getAssetBaseUrl(),
+            baseUrl: req.baseUrl,
```

Comment:
TODO: create a function that generates the front-end url, which returns the base url by default.
This function should be overridable, in order to have projects-specific URL

Reply:
Also, take into account prefix